src/user/user.html

```diff
@@ -15,69 +15,78 @@
+            </div>
 
-      <div layout="row" ng-if="$ctrl.fields && $ctrl.fields.length > 0" ng-repeat="field in $ctrl.fields">
-        <label flex="10" class="field-label">{{field.label}}</label>
-        <md-input-container class="field-input-container">
-          <input ng-if="!field.values || field.values.length == 0"  ng-model="$ctrl.user.customFields[field.key]" type="text" ng-required="field.required">
-          <md-select style="padding-bottom: 2em" ng-if="field.values && field.values.length > 0" ng-model="$ctrl.user.customFields[field.key]" ng-required="field.required">
-            <md-option ng-value="value" ng-repeat="value in field.values">{{ value }}</md-option>
-          </md-select >
-        </md-input-container>
-      </div>
+            <div layout="row" ng-if="$ctrl.fields && $ctrl.fields.length > 0" ng-repeat="field in $ctrl.fields">
+              <label flex="10" class="field-label">{{field.label}}</label>
+              <md-input-container class="field-input-container">
+                <input ng-if="!field.values || field.values.length == 0"  ng-model="$ctrl.user.customFields[field.key]" type="text" ng-required="field.required">
+                <md-select style="padding-bottom: 2em" ng-if="field.values && field.values.length > 0" ng-model="$ctrl.user.customFields[field.key]" ng-required="field.required">
+                  <md-option ng-value="value" ng-repeat="value in field.values">{{ value }}</md-option>
+                </md-select >
+              </md-input-container>
+            </div>
+          </div>
 
-    </md-card-content>
-    <div>
-      <md-button class="md-raised" ng-click="$ctrl.generateToken()">
-        Generate new personal access token
-      </md-button>
-      <div class="gravitee-user-token">Tokens you have generated that can be used to access the Gravitee.io management API.</div>
+          <div>
+            <gravitee-image image-id="$ctrl.user.id" image="$ctrl.user.picture" image-url="$ctrl.user.picture_url"
+                            image-border-radius="true"
+                            image-name="$ctrl.user.displayName"
+                            image-form="formUser"
+                            can-delete="true" delete-label="Use default"></gravitee-image>
+          </div>
+        </div>
+        <div layout="row" layout-align="end end" style="margin-top: 20px;">
+          <md-button type="submit" ng-disabled="formUser.$pristine">Update</md-button>
+          <md-button ng-click="$ctrl.cancel()" ng-disabled="formUser.$pristine">Cancel</md-button>
+        </div>
+      </form>
+    </div>
+  </div>
 
+  <div class="gv-form">
+    <h2>Personal access tokens</h2>
+    <span>Tokens you have generated that can be used to access the Gravitee.io management API.</span>
+    <div class="gv-form-content" layout="column">
+      <md-button class="md-raised" ng-click="$ctrl.generateToken()" style="width: 40%; margin: 0 auto;">
```

Comment:
Don't know if the final view has been validated by APIM team but the "generate" button isn't align to the left whereas it was in 1.30. 
Removing the margin here get the same rendering.